packages/pf4-component-mapper/package.json

```diff
@@ -64,8 +64,9 @@
   },
   "dependencies": {
     "@data-driven-forms/common": "*",
+    "downshift": "^5.4.3",
     "prop-types": "^15.7.2",
-    "downshift": "^5.4.3"
+    "react-jss": "^10.5.0"
```

Comment:
this dependency is not necessary. You are using just CSS

Reply:
oh right, leftover